src/sass/file-upload-input/file-upload-input.scss

```diff
@@ -0,0 +1,96 @@
+@import "../variables/variables";
+
+div.file-upload-input {
+    display: grid;
+    grid-template-rows: 1fr;
+    grid-template-columns: 1fr;
+    grid-template-areas: "center";
+}
+
+div.file-upload-input label[for="file-upload-input__input"] {
+    grid-area: center;
+}
+
+input.file-upload-input__input[type="file"] {
+    grid-area: center;
+    border-radius: var(--border-radius-100, 16px);
+}
+
+div.file-upload-input__container {
+    display: flex;
+    flex-direction: row;
+    background: var(--color-background-secondary, #f7f7f7);
+    border: 1px dashed var(--color-neutral-500, #8f8f8f);
+    border-radius: var(--border-radius-100, 16px);
+    min-width: fit-content;
+    padding: var(--spacing-200, 16px);
+    gap: var(--spacing-100, 8px);
+}
+
+div.file-upload-input__upload-icon {
+    background: var(--color-background-primary, #fff);
```

Comment:
Ideally we shouldn't have defaults for var tokens. 
The reason is if tokens aren't loaded, we want the page to break, so that tokens will be loaded. Also if this token is updated, the default no longer works.